configuration/resize-disk-image.md

```diff
@@ -75,8 +75,8 @@ Done.
 
 If you want install a lot of software in your TemplateVM, you may need to increase the amount of disk space your TemplateVM can use.
 
-1.  Make sure that all the VMs based on this template are shut off (including netvms etc).
-2.  Sanity check: verify that none of loop device are pointing at this template root.img: `sudo losetup -a`
+1.  Make sure that all the VMs based on this template are powered off (including netvms etc).
```

Comment:
I prefer something that isn't a negation. I see in the same section there is 'shut off'. What about 'shut down' since I think that appears elsewhere in the docs frequently?